frontend/src/services/api.js

```diff
@@ -1,14 +1,15 @@
 import axios from "axios";
+import auth from "@/store/modules/auth";
 
 const api = axios.create({
     baseURL: "http://localhost:8000/api", //tu najlepiej żeby to dało się zmienić w jednym miejscu najlepiej żeby można to było ustawić w czasie buildu
     timeout: 5000,
     headers: {
         "Content-Type": "application/json",
+        "Authorization": auth.state.accessToken === undefined ? "" : "Bearer " + auth.state.accessToken,
     },
 });
```

Comment:
![obraz](https://user-images.githubusercontent.com/68618094/207021407-795ec9ef-a876-454e-8bf0-35ad2edd8144.png)
To sprawia, że użytkownik ani gość nie są w stanie zobaczyć żadnego katalogu